bugbug/utils.py

```diff
@@ -280,14 +294,43 @@ def create_tar_zst(path: str) -> None:
     if not os.path.exists(inner_path):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), inner_path)
 
-    subprocess.run(["tar", "-I", "zstdmt", "-cf", path, inner_path], check=True)
+    try:
+        subprocess.run(
+            ["tar", "-I", "zstdmt", "-cf", path, inner_path],
+            check=True,
+            stderr=subprocess.PIPE,
+        )
+    except subprocess.CalledProcessError as error:
+        if (
+            error.returncode == 2
+            and "zstdmt: not found" in str(error.stderr, "utf-8").strip()
+        ):
+            with open_tar_zst(path, "w") as tar:
+                tar.add(inner_path)
+        else:
+            raise error
 
 
 def extract_tar_zst(path: str) -> None:
     if not os.path.exists(path):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), path)
 
-    subprocess.run(["tar", "-I", "zstdmt", "-xf", path], check=True)
+    try:
+        subprocess.run(
+            ["tar", "-I", "zstdmt", "-xf", path],
+            check=True,
+            stderr=subprocess.PIPE,
+        )
+    except subprocess.CalledProcessError as error:
+        if (
+            error.returncode == 2
+            and "zstdmt: Cannot exec: No such file or directory"
+            in str(error.stderr, "utf-8").strip()
+        ):
```

Comment:
I think the exact error message and maybe even the error code could differ based on the platform. I would avoid relying on such assumptions and try using the Python API anyway. 

Also, it will be helpful here to log a warning to mention that we are falling back to `zstandard`, which could be slower.